Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -12,10 +12,19 @@
   license: "CC-BY-SA-4.0"
   copyright: "Made by JustInvoke freesound.org"
   source: "https://freesound.org/people/JustInvoke/sounds/446100/"
- 
+
 - files:
   - jesterstep1.ogg
   - jesterstep2.ogg
   license: "CC-BY-SA-3.0"
   copyright: "Made and posted by GentleJester#8754 on the SS14 discord."
   source: "https://discord.com/channels/310555209753690112/311537926376783886/1097222920813674527"
+
+- files:
+  - heavystep1.ogg
+  - heavystep2.ogg
+  - heavystep3.ogg
+  - heavystep4.ogg
+  license: "CC-BY-3.0"
+  copyright: "Taken from https://mp3crown.org. Converted from MP3 to OGG and downgraded to -7db"
+  source: "https://mp3crown.org/downloadmp3/oeh0x71lkbx_heavy-footstep-sound-effect-hd-hq.html"
```

Comment:
So we think it would be best that the source sound is used that has the license. That is trackable whereas the other two are not.